tt_metal/hw/ckernels/grayskull/llk_lib/llk_pack_common.h

```diff
@@ -138,6 +184,74 @@ inline void llk_pack_relu_config(std::uint32_t config) {
     TTI_NOP; TTI_NOP;
 }
 
-inline void llk_pack_reconfig_data_format(std::uint32_t new_operand) {
-   reconfig_packer_data_format(new_operand);
+inline void _llk_pack_reconfig_l1_acc_(const std::uint32_t enable) {
+    TT_LLK_DUMP("llk_pack_reconfig_l1_acc({})", enable);
+}
+
+template <bool untilize = false, ReduceDim dim>
+inline void _llk_pack_reduce_mask_config_() {
+    // More information about the configuration can be read in B0 llk_pack_common.h
+    // The only difference is that on GS we cannot configure which packer uses which
+    // TILE_ROW_SET_MAPPING[0:3] register; the mapping is 1:1
+    uint32_t edge_offset_sec1_mask = 0xffff;
+
+    // Wait for packer to finish to avoid breaking its current configuration
+    TTI_STALLWAIT(p_stall::STALL_CFG, p_stall::PACK);
+
+    if constexpr (dim == ReduceDim::REDUCE_ROW) {
+        edge_offset_sec1_mask = 0x0001;
+        // Configure TILE_ROW_SET_MAPPING registers
+        if constexpr (untilize) {
+            TTI_SETDMAREG(0, 0x1111, 0, LO_16(p_gpr_pack::TMP0));
+            TTI_SETDMAREG(0, 0x1111, 0, HI_16(p_gpr_pack::TMP0));
+
+            TTI_WRCFG(p_gpr_pack::TMP0,  p_cfg::WRCFG_32b, TILE_ROW_SET_MAPPING_0_row_set_mapping_0_ADDR32);
+            TTI_WRCFG(p_gpr_pack::TMP0,  p_cfg::WRCFG_32b, TILE_ROW_SET_MAPPING_1_row_set_mapping_0_ADDR32);
+            TTI_WRCFG(p_gpr_pack::TMP0,  p_cfg::WRCFG_32b, TILE_ROW_SET_MAPPING_2_row_set_mapping_0_ADDR32);
+            TTI_WRCFG(p_gpr_pack::TMP0,  p_cfg::WRCFG_32b, TILE_ROW_SET_MAPPING_3_row_set_mapping_0_ADDR32);
+        } else {
```

Comment:
Not following why this is templatized on untilize. If it's just for the sake of bringing this up to date with latest buda backend, that's fine I guess.

Reply:
yes this is common with buda, but if its not used we will clean it up there too